### DISCUSSION TITLE
iris without sodium

### DISCUSSION THREAD

**Discussion by TTOMTTIM** [IrisShaders/Iris#692] [Q&A] [answered]
I want to install iris mod without it containing sodium but when I turn on installer I don't have the option to do it and only option is iris & sodium

**Comment by kapcer69** [comment]
I have a similar question... I mean can I just put it in the mods folder if I've already installed normal Sodium?

**Reply by BeansBeefBroccoli** [reply]
I don't think you can, but why would you need to? Sodium is included in Iris. If it's a matter of another mod depending on Sodium, you can use 7zip or a similar program to open the .jar file of the other mod and edit fabric.mod.json to not depend on Sodium. That should work I think if that's your problem.

**Comment by NoComment1105** [comment, marked as answer]
Iris currently requires a specific fork of sodium to work, though if you would like the mods separated you can compile each of them separately from their respective repositories

**Reply by BeansBeefBroccoli** [reply]
@NoComment1105, the fork just got archived.
